src/_locales/en/messages.json

```diff
@@ -3,7 +3,7 @@
         "message": "Facebook Container isolates your Facebook activity from the rest of your web activity in order to prevent Facebook from tracking you outside of the Facebook website via third party cookies.",
         "description": "Description of the extension. DO NOT TRANSLATE \"Facebook Container\"."
     },
-    "onFacebookHeader": {
+    "on-facebook-subhead": {
```

Comment:
IIRC we don't like to change message strings once they've been published to Pontoon. These should probably be moved to whole new message IDs.